recipes/bdwgc/all/patches/update-cmake-8_0_6.patch

```diff
@@ -0,0 +1,821 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -21,232 +21,592 @@
+ #  this will generate gc.sln
```

Comment:
the diff is pretty huge. did you take the patch somewhere? it might be problematic to maintain it in a long term...

Reply:
update-cmake-8_0_6.patch is similar in size to update-cmake.patch (corresponding to v8.0.4)
The future upstream version (v8.2.2) won't require any patching of CMakeLists.txt.

Reply:
For the review purpose, please do: diff -ru update-cmake.patch update-cmake-8_0_6.patch
The diff contains only ~100 lines.

Reply:
thank you for the clarification